Cargo.toml

```diff
@@ -53,3 +53,9 @@ opt-level = "s" # optimize for size
 
 [profile.release.package.datadog-serverless-trace-mini-agent]
 strip = true
+
+[profile.fuzz]
+inherits = "dev"
+opt-level = 3
+incremental = false
+codegen-units = 1
```

Comment:
Did you craft these yourself or follow guidelines from somewhere? Basically, can I read up somewhere on these choices?

Reply:
https://camshaft.github.io/bolero/library-installation.html  I'll add a comment